src/Auth0.ManagementApi/Clients/ClientGrantsClient.cs

```diff
@@ -28,7 +32,7 @@ public ClientGrantsClient(IApiConnection connection)
         /// <returns>The new <see cref="ClientGrant"/> that has been created.</returns>
         public Task<ClientGrant> CreateAsync(ClientGrantCreateRequest request)
         {
-            return Connection.PostAsync<ClientGrant>("client-grants", request, null, null, null, null, null);
+            return Connection.SendAsync<ClientGrant>(HttpMethod.Post, BuildUri("client-grants"), request);
```

Comment:
**Nit:** it looks like `"client-grants"` is used enough that maybe a class-level constant would be useful here? Or maybe add to the base URI in the constructor?

Reply:
Would make code-gen more confusing and is going to be replaced by that.